tests/apps/search/test_indexers_subjects.py

```diff
@@ -18,38 +20,32 @@ class SubjectsIndexersTestCase(TestCase):
     and especially dynamic mapping shape in ES
     """
 
-    @responses.activate
-    def test_indexers_subjects_get_data_for_es(self):
+    @mock.patch.object(
+        Picture, "img_src", new_callable=mock.PropertyMock, return_value="123.jpg"
+    )
+    def test_indexers_subjects_get_data_for_es(self, _mock_picture):
         """
-        Happy path: the data is fetched from the API properly formatted
+        Happy path: the data is fetched from the models properly formatted
         """
-        responses.add(
-            method="GET",
-            url=settings.SUBJECT_API_ENDPOINT + "?page=1&rpp=50",
-            match_querystring=True,
-            json={
-                "count": 51,
-                "results": [
-                    {"id": 62, "image": "example_cs.png", "name": "Computer Science"}
-                ],
-            },
+        subject1 = SubjectFactory(
+            page_title={"en": "my first subject", "fr": "ma première sujet"},
```

Comment:
Using `première` to check it works with accents?

Reply:
hmm more liike a copy/pasta failure from `organizations"... Maybe I can write `ma première thématique`...